src/components/SUE/report/SueReportLocation.tsx

```diff
@@ -255,7 +254,7 @@ export const SueReportLocation = ({
               setSelectedPosition(nativeEvent.coordinate);
 
               try {
-                await reverseGeocode(nativeEvent.coordinate);
+                await handleGeocode(nativeEvent.coordinate);
```

Comment:
is this change intended here with this PR?

Reply:
I have seen this error and wanted to correct it.

Reply:
ok yes, I already thought so